magic_spoiler/__main__.py

```diff
@@ -373,7 +373,7 @@ def write_cards(
         if pow_tough:
             card_xml_file.write("<pt>" + pow_tough + "</pt>\n")
 
-        if "loyalty" in card.keys():
+        if str(card.get("loyalty")) != "None":
```

Comment:
this is bad python code

Reply:
That is very much possible. :)

Thank you for appending and improving it in #255!

Reply:
you're welcome